subsystems/drivebase/SwerveDrive.java

```diff
@@ -311,8 +327,24 @@ public void mustangPeriodic() {
         Logger.recordOutput(DRIVEBASE_PITCH, getPitch());
         Logger.recordOutput(DRIVEBASE_ROLL, getRoll());
 
+       
+
+        if (kConfig.kDriveMotorType.equals(Motor_Type.KRAKEN_X60)) {
+            emaCurrent = currentAlpha * ((TalonFX) mModules[0].getDriveMotor()).getSupplyCurrent() + (1 - currentAlpha) * emaCurrent; // weighted current average of front-right drive motor
```

Comment:
My biggest problem with this approach is that I think it's measuring the wrong thing. We should be measuring the total current from the PDH. That said, as long as we continue to not trust the PDH, this is probably the next best thing as the drive motors dominate current draw.

On this approach:
1. why are we grabbing this from just the single motor instead of all of them? They're not all necessarily doing the same thing
2. having to check the motor type and cast to a TalonFX indicates that our interface is wrong - we ought to be able to hide this behind the interface. Though since the NEO doesn't let you find out the supply current, that implementation would probably be returning 0, which is no better or worse than doing nothing if it's not a kraken
3. do we not set the motor type per module? Seems like we probably should in case we run into a situation where we run out of krakens or something. But that's probably a different PR

Reply:
Just wondering, but wouldn't a drivebase with some krakens and some neos skew when at max speed or at least behave wierdly

Reply:
Probably. But it would be better than not competing at all